src/components/Layout.js

```diff
@@ -26,13 +26,13 @@ const TemplateWrapper = ({ children }) => {
         <link
           rel="icon"
           type="image/png"
-          href={`${withPrefix("/")}img/favicon-32x32.png`}
+          href={`${withPrefix("/")}src/img/icon.png`}
```

Comment:
This does not seem to exist either: https://ironicbaremetal.org/img/icon.png. The ones with sizes do exist: https://ironicbaremetal.org/icons/icon-48x48.png?v=07155758fa26744f101f5d047bd95b9c

Reply:
It's a weird one.  The gastbyconfig.js settings are correct, but it's not building properly and associating it with all the various sizes.  Give me a bit to try some things out.